src/cmd/turnkey/pkg/encrypt.go

```diff
@@ -0,0 +1,95 @@
+package pkg
+
+import (
+	"encoding/hex"
+	"encoding/json"
+
+	"github.com/rotisserie/eris"
+	"github.com/spf13/cobra"
+
+	"github.com/tkhq/go-sdk/pkg/enclave_encrypt"
+)
+
+var (
+	// user is the user ID to import wallets and private keys with.
+	user string
+
+	// Filepath to write the import bundle to.
+	importBundlePath string
+
+	// Filepath to read the encrypted bundle from.
+	encryptedBundlePath string
+
+	// Filepath to read the plaintext from that will be encrypted.
+	plaintextPath string
+)
+
+func init() {
+	encryptCmd.Flags().StringVar(&importBundlePath, "import-bundle-path", "/import_bundle.txt", "filepath to write the import bundle to.")
+	encryptCmd.Flags().StringVar(&encryptedBundlePath, "encrypted-bundle-path", "/encrypted_bundle.txt", "filepath to read the encrypted bundle from.")
+	encryptCmd.Flags().StringVar(&plaintextPath, "plaintext-path", "", "filepath to read the plaintext from that will be encrypted.")
+
+	rootCmd.AddCommand(encryptCmd)
+}
+
+var encryptCmd = &cobra.Command{
+	Use:   "encrypt",
+	Short: "Encrypt a plaintext",
+	Long:  `Encrypt a plaintext into a bundle to be imported to a Turnkey secure enclave.`,
+	PreRun: func(cmd *cobra.Command, args []string) {
+		if plaintextPath == "" {
+			OutputError(eris.New("Filepath for plaintext must be specified"))
+		}
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		// read from import bundle path
+		importBundle, err := readFile(importBundlePath)
+		if err != nil {
+			OutputError(err)
```

Comment:
we don't need a `return` here and below? It's not clear at a glance that this exits the control flow

```
return OutputError(err)
```

Reply:
from their docs: `OutputError prints an error to the console and exits.`

also even if wanted to add `return`, it won't compile bc OutputError(err) has no value and can't be used as a return value